api/nakadi-event-bus-api.yaml

```diff
@@ -317,7 +659,9 @@ paths:
           required: true
         - name: batch_limit
           in: query
-          description: Maximum number of `SimpleStreamElement`s in each chunk of the stream. If unspecified assumes default value 1 (i.e. each event is individually submitted).
+          description: |
+            Maximum number of `EventStreamBatch`s in each chunk of the stream. If unspecified assumes default
+            value 1 (i.e. each event is individually submitted).
```

Comment:
This is for sending events out of Nakadi, yes? In this case probably one should talk about _delivering_ events and not _submitting_ them.